template.adoc

```diff
@@ -49,21 +49,34 @@ You'll explore how to....
 // You'll next add the various sections that are needed for a particular guide.
 
 
-== Getting Started
 // Add this getting started section to your guide if it is applicable. Use the following include to pull in the git clone instructions from the guides-common repo.
-
+== Getting started
 include::{common-includes}/gitclone.adoc[]
 
+// This is a subsection of the "Getting started" section above. It should briefly walk the user through
+// how to setup everything in the "finish" directory and try out the finished version of what
+// they will be building.
+=== Try what you'll build
+

```

Comment:
I still think Try what you'll build could be improved:
Try it out? View the end result? See what you will build? See it first? I know we were talking about this in our first call, not sure what was decided.

Reply:
Personally, I think "Try what you'll build" makes the most sense. Its broad enough to include everything in the guide, but not too broad to create confusing. Its also not specific enough to single out parts of the guide.